cmd/thv-operator/Taskfile.yml

```diff
@@ -135,21 +135,16 @@ tasks:
         sh: KO_DOCKER_REPO=kind.local ko build --local -B ./cmd/thv-operator | tail -n 1
       TOOLHIVE_IMAGE:
         sh: KO_DOCKER_REPO=kind.local ko build --local -B ./cmd/thv-proxyrunner | tail -n 1
-      REGISTRY_API_IMAGE:
```

Comment:
maybe we don't have to build it here, but for sure to load it in kind (and ensure renovate is changing it to the latest version, or just use the `latest` tag)

Reply:
This was implemented in commit 438c810e! The Taskfile now:
1. Pulls the registry API image: `docker pull {{.REGISTRY_API_IMAGE}}`
2. Loads it into kind: `kind load docker-image --name toolhive {{.REGISTRY_API_IMAGE}}`
3. Configures it in the operator: `--set registryAPI.image={{.REGISTRY_API_IMAGE}}`

Default is `ghcr.io/stacklok/thv-registry-api:latest` (overridable via `REGISTRY_API_IMAGE` env var for testing other versions).

See `cmd/thv-operator/Taskfile.yml:144-146, 152`